testapi.pm

```diff
@@ -620,13 +624,19 @@ I<Make sure the command does not write to the serial output.>
 
 =cut
 sub assert_script_run {
-    my ($cmd, $timeout, $fail_message) = @_;
+    my ($cmd, $args, $fail_message) = @_;
+    unless (ref $args eq ref {}) {
```

Comment:
Ok, but can you change it to comparing to HASH? We already use it that way. See [Schema::Result::Jobs#L262](https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Schema/Result/Jobs.pm#L262) or [Scheduler::Scheduler#L625](https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Scheduler/Scheduler.pm#L625) or [WebSockets::Server#L165](https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebSockets/Server.pm#L165)